src/components/Person/Person.jsx

```diff
@@ -1 +1,42 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age = '',
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const isAge = age && age > 0;
+
+  const married = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  const isPartner = isMarried
+    ? `${partnerName} is my ${married}`
+    : 'I am not married';
```

Comment:
```suggestion
  const marriageInfo = isMarried
    ? `${partnerName} is my ${married}`
    : 'I am not married';
```

Reply:
isPartner - is name for boolean variable that will tell if someone is partner for anyone